rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,134 @@
+
+The badges below are generated using https://shields.io/ and link to a placeholder wiki page which will later contain explanations of each status.
+
+We definitely want badges for stability, possibly with sub-badges for convergence status:
+
+- [![in development](https://img.shields.io/badge/stability-in%20development-orange)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![preview](https://img.shields.io/badge/stability-preview-yellow)](https://github.com/microsoft/fluentui/wiki/Package-status)
+  - Possible sub-badge: [![preview (legacy)](<https://img.shields.io/badge/stability-preview%20(legacy)-yellow>)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![stable](https://img.shields.io/badge/stability-stable-green)](https://github.com/microsoft/fluentui/wiki/Package-status)
+  - Possible sub-badge: [![stable (legacy)](<https://img.shields.io/badge/stability-stable%20(legacy)-green>)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![deprecated](https://img.shields.io/badge/stability-deprecated-red)](https://github.com/microsoft/fluentui/wiki/Package-status)
+
+We might also want separate badges for category/convergence/??? (naming suggestions welcome)
+
+- [![legacy](https://img.shields.io/badge/convergence-legacy-lightgrey)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![mixed](https://img.shields.io/badge/convergence-mixed-yellow)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![done](https://img.shields.io/badge/convergence-done-green)](https://github.com/microsoft/fluentui/wiki/Package-status)
+
+Notes:
+
+- Charting and maybe experiments would probably fall into the "preview (legacy)" category
+- Suite packages should list which components are converged or not
+
+### Add `@fluentui/react-preview` and related communication channels
+
+Sometime in the next couple months after version 8 release, add a `@fluentui/react-preview` package which exports _only_ converged components which have reached "80% ready"/preview status. Notes:
+
+- Exports will go away once they're moved to the suite
+- Document clearly from the start that the stability is lower, and people can consume it if they're willing to accept that minor versions may sometimes contain breaking changes (to be clearly noted in the changelogs)
+- Either 1st or 3rd parties can use this if they're willing to accept the extra risk/work
+- Readme should be kept updated with status of each component
+- If people prefer, they can use the individual preview component packages, but `@fluentui/react-preview` is the "one-stop shop" for previewing converged components
+
+We should also make a private Teams channel within Fluent Community for internal partners who are working closely with us to try out the preview components. (It's possible that this could be expanded to external partners in the future.) This would be a very direct way for partners to share feedback and get support with preview components.
```

Comment:
Love the idea of the Teams channel for internal partners we are working closely with